pom.xml

```diff
@@ -20,21 +20,82 @@
 	</scm>
 	<properties>
 		<java.version>1.7</java.version>
+		<rxjava.version>1.1.5</rxjava.version>
 		<spring-boot.version>1.4.1.BUILD-SNAPSHOT</spring-boot.version>
+		<spring.tuple.version>1.0.0.RELEASE</spring.tuple.version>
+		<spring.integration.tuple.version>1.0.0.RELEASE</spring.integration.tuple.version>
+		<reactor.version>3.0.0.BUILD-SNAPSHOT</reactor.version>
 	</properties>
 	<dependencyManagement>
 		<dependencies>
 			<dependency>
 				<groupId>org.springframework.cloud</groupId>
-				<artifactId>spring-cloud-stream-dependencies</artifactId>
+				<artifactId>spring-cloud-stream</artifactId>
 				<version>1.1.0.BUILD-SNAPSHOT</version>
-				<type>pom</type>
-				<scope>import</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-stream-build-tools</artifactId>
+				<version>1.1.0.BUILD-SNAPSHOT</version>
```

Comment:
No, because we'd want to use it for binders so that they get some dependencies managed like that - e.g. as parent. Having `${project.version}` there would mean that if the child has a different version, that value will be used as a placeholder.